File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
  let sum1 = 0
  let sum2 = 0

  for (let i = 0; i < num_list.length; i++) {
    if(i % 2 == 0) sum1 += num_list[i]
    if(i % 2 == 1) sum2 += num_list[i]
  }

  if (sum1 > sum2) return sum1
  if (sum2 > sum1) return sum2
  if (sum1 == sum2) return sum1
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
  public int solution(int[] num_list) {
    int odd = 0;
    int even = 0;

    for (int i = 1; i <= num_list.length; i++) {
      if (i % 2 == 0) even += num_list[i - 1];
      else odd += num_list[i - 1];
    }

    return Math.max(odd, even);
  }
}